cargo-dist/src/config.rs

```diff
@@ -602,6 +602,8 @@ pub enum InstallerStyle {
     Homebrew,
     /// Generate an msi installer that embeds the binary
     Msi,
+    /// Generate a docker image for x86_64-unknown-linux-gnu
```

Comment:
Is the only limit what's available in CI right now?

Reply:
kinda a semantic awkwardness that i'm not sure how to resolve:

* want to be able to generate docker images for multiple platforms
* don't want to have "turning on the docker installer" generate a million docker images

simplest solution i could think of was "it's per-platform but only one platform supports it until we have better answers"...